VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -149,7 +149,6 @@ public void onException(Exception ex) {
                 buttonAddComment.setEnabled(true);
             }
         };
-        createCommentTask.setTaskProcessCallback(null);
```

Comment:
Actually, upon rethinking it, I guess the two can technically go together, so we might be better off doing this manually.